docs/examples/organizational-units/division-code-extension/extension.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": {
+    "en": "Division codes"
+  },
+  "description": {
+    "en": "Adds a divisionCode field to the Organization building block"
+  },
+  "documentationUrl": {
+    "en": "http://standard.open-contracting.com"
```

Comment:
Do we need some way to indicate that in real-life this should point to specific documentation, NOT the standard docs (I realise this is not the standard docs URL!) - maybe an alternative fictional URL like "http://github.com/example_publisher/ocds_division_code_extension"?

Reply:
That makes sense, I'll change it.